package errors

import "testing"

func TestError(t *testing.T) {
	t.Log("a")
}